### DISCUSSION TITLE
No one promise resolved

### DISCUSSION THREAD

**Discussion by redn3ck** [1715173329/UnblockNeteaseMusic#1081] [Q&A]
### Bug 描述

《乌梅子酱》等一些歌曲均提示`No one promise resolved` 

### 预期行为

使用`release unblockneteasemusic-linux-x64` 则提示 `All promises were rejected`
![图片](https://user-images.githubusercontent.com/12512242/234518952-feeb7ecc-d3dd-4dc4-a3de-e01a55c9aa55.png)

我感觉是同样的问题，但排查无果，周杰伦的歌曲就正常解析

### 日志内容

![图片](https://user-images.githubusercontent.com/12512242/234518029-43c3d2a2-adcb-4fc6-a6cc-a5a06a810d0d.png)

### 网易云音乐版本号

2.10.8

### 操作系统

ubuntu，香港云

### 问题排查

- [X] 我确认我使用的核心是由 UnblockNeteaseMusic 项目官方发行，不是其他任何 fork。
- [X] 我确认我已经升级到了最新的核心版本（推荐使用最新构建而不是 release）。
- [X] 我确认我已经启用了 HTTPS 端口。
- [X] 我确认我已经正确设置了 EndPoint。
- [X] 我确认我已经在对应的客户端正确安装了 CA 证书。

**Comment by 1715173329** [comment]
受区域限制，非大陆地区的请求可能会出现问题。这首歌网易云音乐本身就有版权，开启 pyncmd 音源即可。
